clients/lcdproc/main.c

```diff
@@ -423,8 +438,9 @@ process_configfile(char *configfile)
 		islow = config_get_int(progname, "Delay", 0, -1);
 	}
 
-	if ((tmp = config_get_string(progname, "DisplayName", 0, NULL)) != NULL)
-		displayname = strdup(tmp);
```

Comment:
What about this? Why is this code removed?

Reply:
Sorry, my bad. Added again